client/routes.js

```diff
@@ -47,7 +47,7 @@ class Routes extends Component {
           </Switch>
         }
         {/* Displays our Login component as a fallback */}
-        <Route component={Login} />
+        <Route component={Products} />
```

Comment:
Why did you have to change this? If a visitor tries to go to a page it cannot access, it will direct them to login